src/common/bitstream_test.cpp

```diff
@@ -119,6 +121,88 @@ void testBitstreamReadBits() {
     }
 }
 
+void testBitstreamReadQuantitizedFloat() {
+    static std::vector<uint8_t> expectedBuf = std::vector<uint8_t>({
+        0x6C, 0x2D, 0x76, 0x55, 0x35, 0xCA, 0x16 //6C2D7 65535 CA16
+    });
+    BitStream bitstream(expectedBuf);
+
+    float x;
+    float y;
+    float z;
+    bitstream.readQuantitizedFloat(x, 20, 8192.0f);
+    bitstream.readQuantitizedFloat(y, 20, 8192.0f);
+    bitstream.readQuantitizedFloat(z, 16, 1024.0f);
+    assertEqual( compareFloats(x, 3674.85f, 100), 0);
+    assertEqual( compareFloats(y, 2726.79f, 100), 0);
+    assertEqual( compareFloats(z, 91.1576f, 100), 0);
+}
+
+void testBitstreamWriteQuantitizedFloat() {
```

Comment:
Yeah, I'm just thinking that there are 3 kinds of tests here: 1) that reading data produces expected float values, 2) that writing float values produces expected data, and 3) that reading data to floats and then writing those floats produces the data you started with (aka nothing 'lost' when reading + writing). The write tests you have are more like the 3rd case, but I suppose that transitively covers the 2nd case - so as long as the read test passes, and this read-then-write test passes, you can assume that write is working. So perhaps it's good as-is.